database/src/main/kotlin/no/nav/su/se/bakover/database/revurdering/RevurderingPostgresRepo.kt

```diff
@@ -129,6 +142,196 @@ internal class RevurderingPostgresRepo(
         }
     }
 
+    internal fun lagre(revurdering: StansAvYtelseRevurdering, sessionContext: TransactionContext) {
+        sessionContext.withSession { tx ->
+            when (revurdering) {
+                is StansAvYtelseRevurdering.SimulertStansAvYtelse -> {
+                    """
+                    insert into revurdering (
+                        id,
+                        opprettet,
+                        periode,
+                        simulering,
+                        saksbehandler,
+                        revurderingsType,
+                        vedtakSomRevurderesId,
+                        årsak,
+                        begrunnelse,
+                        attestering,
+                        skalFøreTilBrevutsending
+                    ) values (
+                        :id,
+                        :opprettet,
+                        to_json(:periode::json),
+                        to_json(:simulering::json),
+                        :saksbehandler,
+                        :revurderingsType,
+                        :vedtakSomRevurderesId,
+                        :arsak,
+                        :begrunnelse,
+                        to_json(:attestering::json),
+                        :skalFoereTilBrevutsending
+                    ) on conflict(id) do update set
+                        periode=to_json(:periode::json),
+                        simulering=to_json(:simulering::json),
+                        saksbehandler=:saksbehandler,
+                        revurderingsType=:revurderingsType,
+                        vedtakSomRevurderesId=:vedtakSomRevurderesId,
+                        årsak=:arsak,
+                        begrunnelse=:begrunnelse
+                    """.trimIndent()
+                        .insert(
+                            mapOf(
+                                "id" to revurdering.id,
+                                "opprettet" to revurdering.opprettet,
+                                "periode" to objectMapper.writeValueAsString(revurdering.periode),
+                                "simulering" to objectMapper.writeValueAsString(revurdering.simulering),
+                                "saksbehandler" to revurdering.saksbehandler,
+                                "revurderingsType" to RevurderingsType.SIMULERT_STANS,
+                                "vedtakSomRevurderesId" to revurdering.tilRevurdering.id,
+                                "arsak" to revurdering.revurderingsårsak.årsak.toString(),
+                                "begrunnelse" to revurdering.revurderingsårsak.begrunnelse.toString(),
+                                "attestering" to Attesteringshistorikk.empty().hentAttesteringer().serialize(),
+                                "skalFoereTilBrevutsending" to false,
+                            ),
+                            tx,
+                        )
+                    fradragsgrunnlagPostgresRepo.lagreFradragsgrunnlag(
```

Comment:
Trenger ikke gjøre det nå, men vi kunne kanskje hatt et repo for å lagreVilkårsvurderingerOgGrunnlag for å separert litt mer.

Reply:
Det kunne nok vært hensiktsmessig ja.